cmd/containerboot/main.go

```diff
@@ -16,6 +16,8 @@
 //   - TS_ROUTES: subnet routes to advertise.
 //   - TS_DEST_IP: proxy all incoming Tailscale traffic to the given
 //     destination.
+//   - TS_EGRESS_IP: proxy all incoming non-Tailscale traffic to the given
```

Comment:
This has five different names. 

TS_EGRESS_IP
EgressProxyTo
TailnetProxyTarget
TailscaleProxyTarget
TargetIP


Let's pick one and use that everywhere

Reply:
I've changed all to some version of 'TS_EGRESS_IP'

Reply:
I think `TS_TAILNET_TARGET_IP` is more descript